docs/data-layer-api.md

```diff
@@ -0,0 +1,290 @@
+# Data Layer API Reference
+
+The [Data Layer API](/api-docs/store-management/data-layer-api) is a GraphQL management API that allows you to enable and disable the [Big Open Data Layer](/api-docs/analytics/bodl-for-storefronts) for a store.
+
+**Request URL:** 
+
+`https://api.bigcommerce.com/stores/{store_hash}/graphql`
+
+**Path Parameters:**
+
+| Name | Type | Description | Required |
+| :--- | :--- | :--- | :--- |
+| `store_hash` | string | Unique ID for a store | Yes |
+
+## Authentication
+
+### OAuth scopes
+
+| UI Name | Permission | Parameter |
+| :--- | :--- | :--- |
+| Information & Settings | modify | `store_v2_information` |
+| Information & Settings | read-only | `store_v2_information_read_only` |
```

Comment:
I have not checked how the read-only scope works for this endpoint
Have you tested this case?

Reply:
I just checked it, and it turns out read-only doesn't work for the queries, so I decided to remove it. Thanks for pointing it out 👍